### ISSUE TITLE
Uncrewed planetary exploration contracts

### ISSUE THREAD

**Issue by leudaimon** [KSP-RO/RP-0#566]
Following up discussion on #562, I intend to build a TODO to have interesting satellite contracts for the exploration of other bodies. After putting some thought into this, I think a nice progression would be something like this:

- [ ] First Orbit contracts - similar to the already implemented flybys. 
  * requirements: any of: flyby of the body and orbit of a body in the previous "tier"
  * parameters: in orbit and PeA below flyby limit/2-3?

- [ ] Polar orbit contracts - could be a repeatable, demanding different transmittable scientific payloads
  * requirements: First orbit of the body
  * parameters: have some specified experiment and inclination 90+-5°
  * additional idea: bonus credits if the vessel transmits science from all biomes

- [ ] Low polar orbit contracts - can also be repeatable.
  * requirements: Not sure, First orbit or Polar orbit?
  * parameters: circular orbit, below SpaceLow threshold, inclination 90+-5°
  * also bonus (or maybe even required) transmit science from all biomes

- [ ] Impactor - similar to moon impactor, I'm not sure how to deal with this for gas giants, even tough it has already been done.
  * requirements: any of: first orbit of the body or impactor on a body in the previous "tier"
  * parameters: similar to the moon impactor for rocky bodies, don't know about gas giants
  * I think it makes sense to have this with limited completions for each body
  
- [ ] Lander - similar to moon landing, can be made as a repeatable right away for the other (rocky) bodies.
  * requirements: any of: impactor of the body or lander on a body in the previous "tier"
  * parameters: similar to the repeatable moon impactor
  * both for the moon lander and for the other bodies, I think it would be cool to restrict contracts for the different biomes, selecting an unlanded one each time. Thus, maxcompletions would be the number of biomes of each body.

- [ ] High orbit communication satellite for relay purposes - can be an extension from the Comsats currently in development. *Not necessarily last - actually parallel to the others and not exactly scientific*

Any additional ideas? Anything worth changing? Any problems?

**Comment by NathanKell** [comment]
These look good to me. In addition
* Rover contracts to survey an area using waypoints (steal from CC's science contract pack?)
* Sample return

**Comment by pap1723** [comment]
@leudaimon not all of these were implemented with the new contracts, there are still some good ideas left on here!

**Comment by leudaimon** [comment]
Indeed! The current contracts (especially the repeatables) are very biased towards Earth orbits. It would be nice to implement the polar orbit repeatables with a payload requirement. Should be simple to adapt from the Earth repeatables.

**Comment by DRVeyl** [comment]
Writing these contracts is relatively easy.  But why not let it get handled via ScanSat, which will already strongly nudge the player towards those polar contracts, and also at different altitudes for some of the different scans.  You may already get a small amount of repeatability there, with the 3 different basic scans.

**Comment by siimav** [comment]
What @DRVeyl suggested makes sense. The only possible issue is that scansat contracts can't be made repeatable.
